### DISCUSSION TITLE
BPF´s: (How to bypass traffic from a subnet to a host with a defined tcp port)

### DISCUSSION THREAD

**Discussion by kornflake003-GITHUB** [Security-Onion-Solutions/securityonion#6614] [Unsupported Versions] [answered]
Hi,

We are trying to implement some BPF's to indicate that it does NOT process the traffic coming from a certain subnet to a particular host with a defined tcp port:

Text example in global.sls (inside nids):

- (not src net 134.3.100.0/22 && (dst host 125.2.100.75 && dst port 1433))

It is currently blocking all subnet traffic from 134.3.100.0/22, all traffic to the host 125.2.100.75, and all traffic to the tcp port 1433. But all separately.

Thank you very much for your help¡

**Comment by dougburks** [comment, marked as answer]
Maybe try adjusting the parentheses like this:
```
- not (src net 134.3.100.0/22 && dst host 125.2.100.75 && dst port 1433)
```

For more information, please see:
https://docs.securityonion.net/en/2.3/bpf.html

**Reply by kornflake003-GITHUB** [reply]
Thanks a lot for your support @dougburks , it´s working fine.
